common/scripts/installDb.sh

```diff
@@ -85,6 +85,15 @@ __install_db() {
   else
     __check_connection "$DB_IP"
 
+    local proxy_script_name="configureProxy.sh"
```

Comment:
Note: This will only configure proxy when the env `DB_INSTALLED` is false and we are not installing on the same machine as the of Admiral (which brings up a Docker container instead).